### PR TITLE
Bump s6 ebuilds

### PR DESCRIPTION
**Pull request by atlaua**
Other than bumping the versions, I also updated the ebuilds to EAPI 7, fixed linking bugs which previously occured if only one of the static/static-libs flags was enabled and cleaned up the dependency specs.

### CODE REVIEW

File: dev-libs/skalibs/skalibs-2.7.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+		--libdir=/usr/$(get_libdir)/${PN} \
+		--sysdepdir=/usr/$(get_libdir)/${PN} \
+		--enable-clock \
+		--enable-shared \
+		$(use_enable static-libs static) \
+		$(use_enable ipv6)
+}
+
+src_install() {
+	emake DESTDIR="${D}" install
```
Comment (Zlogene) [inline dev-libs/skalibs/skalibs-2.7.0.0.ebuild]: `default`

Comment (atlaua) [reply dev-libs/skalibs/skalibs-2.7.0.0.ebuild]: You mean I should remove the src_install definition and the doc use flag? I guess that'd be okay considering the docs are only half a megabtye large and the other s6* packages don't have the flag either.

Comment (smaeul) [reply dev-libs/skalibs/skalibs-2.7.0.0.ebuild]: I think the intention was 
```
src_install() {
        default

        use doc && einstalldocs
}
```

Comment (atlaua) [reply dev-libs/skalibs/skalibs-2.7.0.0.ebuild]: In any case, I've already pushed a new version which uses the default src_install (and hence installs the docs unconditionally), and I'm inclined to keep it this way. :)

**Comment by atlaua** [comment]
@gentoo/proxy-maint, could you please merge this PR? It's been sitting here long enough.

I'm the maintainer of most involved packages. skalibs, execline and s6 are officially not maintained by me, but as you can see in the git history, smaeul and me have been taking care of them for about two years now. There's no point in waiting for a maintainer ACK much longer.

**Comment by gyakovlev** [comment]
merged.
I corrected a copyright year for s6-linux-utils to 2018.

**Pull request closed without merging**